### PR TITLE
FT: Node v6

### PR DESCRIPTION
**Pull request by alexandre-merle**
switch to node v6

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -14,7 +17,8 @@
   },
   "homepage": "https://github.com/scality/Arsenal#readme",
   "dependencies": {
-    "ajv": "4.1.5",
+    "ajv": "^4.11.2",
+    "eslint-plugin-react": "^4.3.0",
```
Comment (rahulreddy) [inline package.json]: Why do we need to add this plugin?

Comment (alexandre-merle) [inline package.json]: in npm 3, the peer dependencies are not automatically installed anymore, so we need this plugin :)

File: package.json [hunk 2]
```diff
@@ -14,7 +17,8 @@
   },
   "homepage": "https://github.com/scality/Arsenal#readme",
   "dependencies": {
-    "ajv": "4.1.5",
+    "ajv": "^4.11.2",
```
Comment (rahulreddy) [inline package.json]: Use specific version instead of `^`

**Comment by alexandre-merle** [comment]
@rahulreddy updated, the other depedencies will be pinned when forward port of the 6.4 pull request

File: package.json [hunk 3]
```diff
@@ -16,6 +19,7 @@
   "dependencies": {
     "ajv": "4.1.5",
     "ipaddr.js": "1.2.0",
+    "eslint-plugin-react": "^4.3.0",
```
Comment (rahulreddy) [inline package.json]: Isn't this a devDependency though?

Comment (alexandre-merle) [inline package.json]: right :)

**Comment by rahulreddy** [comment]
@alexandre-merle good to go?

**Pull request merged**